src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,101 @@
-export const Pagination = () => {};
+import classname from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: string[],
+  perPage: number,
+  currentPage: number,
+  onPageChange: (item: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total, perPage, onPageChange, currentPage,
+}) => {
```

Comment:
Provide newline for each prop if their amount is >= 3
```suggestion
export const Pagination: React.FC<Props> = ({
  total, 
  perPage, 
  onPageChange, 
  currentPage,
}) => {
```